rw_fs_record.c

```diff
@@ -0,0 +1,78 @@
+/******************************************************************************
+ * Read/write data structure from/to the file
+ *
+ * Copyright Thinnect Inc. 2020
+ * @license MIT
+ *****************************************************************************/
+#include "loglevels.h"
+#define __MODUUL__ "SNST"
+#define __LOG_LEVEL__ (LOG_LEVEL_SeqNumStorage & BASE_LOG_LEVEL)
+#include "log.h"
+
+#include <stdint.h>
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "fs.h"
+
+#define FILE_INST 0
+#define FILE_ERROR -1
```

Comment:
FILE_ERROR seems like something that could easily conflict with 3rdparty libraries. It is also not exposed in the header, but is referred to in the comments of the header.